scripts/projectView.js

```diff
@@ -19,26 +71,18 @@
       $('#projectSection').append(a.toHtml('#project-template'));
     });
     projectView.handleMainNav();
+    projectView.handleLibraryFilter();
+    projectView.handleLanguageFilter();
   };
 
-  projectView.showLanguages = function(){
-    var langList = Project.allLanguages();
-    console.log(langList);
-    langList.forEach(function(lang){
-      $('#languageList').append('<li>' + lang + '</li>');
+  projectView.show = function(func,  tag, id){
```

Comment:
This method has already been refactored to take in a specific function, the type of HTML element to be created and the id of the HTML element to be appended to.